lore/models.py

```diff
@@ -3,6 +3,7 @@
 from tabulate import tabulate
 import lore.estimators
 import lore.serializers
+from lore.estimators.sklearn import SKLearn
```

Comment:
I'm guessing when we subclass, we can move this out, to avoid extra dependencies